ContextualPopupDecorator/ContextualPopup.module.less

```diff
@@ -0,0 +1,116 @@
+// ContextualPopup.module.less
+//
+@import '../styles/variables.less';
+@import '../styles/mixins.less';
+@import '../styles/skin.less';
+
+.contextualPopup {
+	position: absolute;
+
+	.container {
+		position: absolute;
+		background-clip: padding-box;
+		-webkit-background-clip: padding-box;
+		-moz-background-clip: padding-box;
+		box-sizing: border-box;
+
+		&.none { /* Needed to prevent global class being added in the DOM */
+		}
+
+		&.fixedSize {
+			overflow: hidden;
+		}
+
+		.closeButton {
+			position: absolute;
+			margin: 0;
+
+			.focus({
+				.bg {
+					background-color: transparent;
+					background-image: none;
+				}
+			});
+		}
+	}
+
+	:global(.enact-locale-right-to-left) & {
+		.container {
+			direction: rtl;
+		}
+	}
+
+	.arrow {
+		position: absolute;
```

Comment:
Any reason that you didn't define the width and the height like the following code?
```
height: @agate-contextual-popup-arrow-height;
width: @agate-contextual-popup-arrow-width;
```